.github/workflows/ci_cron_weekly.yml

```diff
@@ -170,6 +170,6 @@ jobs:
             # cython and pyerfa versions in ubuntu repos are too old currently
             pip install -U cython setuptools packaging
             pip install -U --no-build-isolation pyerfa
-            ASTROPY_USE_SYSTEM_ALL=1 pip3 install -v --no-build-isolation -e .[test]
+            pip3 install -v --no-build-isolation -e .[test]
```

Comment:
@saimn , like this?

Reply:
The jobs passed but how to be sure the bundled lib is being used?

Reply:
Compilation logs show that the bundled version is used (see DBYTEORDER and include paths):
```
    s390x-linux-gnu-gcc -fno-strict-overflow -Wsign-compare -DNDEBUG -g -O2 -Wall -fPIC -DHAVE_EXPAT_CONFIG_H=1 -DBYTEORDER=4321 -DHAVE_UNISTD_H -Iastropy/utils/xml/src -Icextern/expat/lib -I/home/runner/work/astropy/astropy/tests/include -I/usr/include/python3.13 -c astropy/utils/xml/src/iterparse.c -o /tmp/tmpexq9nhc0.build-temp/astropy/utils/xml/src/iterparse.o
```
so it seems to work.